R/with.R

```diff
@@ -117,6 +120,23 @@ linters_with_tags <- function(tags, ..., packages = "lintr", exclude_tags = "dep
   modify_defaults(..., defaults = tagged_linters)
 }
 
+#' Create a linter configuration based on all available linters
+#'
+#' @inheritParams linters_with_tags
+#'
+#' @examples
+#' names(all_linters())
+#'
+#' @seealso
+#' - [linters_with_defaults] for basing off lintr's set of default linters.
+#' - [linters_with_tags] for basing off tags attached to linters, possibly across multiple packages.
+#' - [available_linters] to get a data frame of available linters.
+#' - [linters] for a complete list of linters available in lintr.
+#' @export
+all_linters <- function(packages = "lintr") {
```

Comment:
This needs ellipsis to allow for configuration.
For example compare
`linters_with_tags(NULL, line_length_linter = NULL)`.

Reply:
Done.